Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:18
```

Comment:
This image actually takes up a lot of space, mainly because this specific image is Node running on top of Debian. Instead, we should use an image such as `node:18-alpine`, which takes up less space.